docker/test.js

```diff
@@ -0,0 +1,279 @@
+var fs = require('fs');
```

Comment:
How can I run the test locally?  Is there any way to run the nginx script locally for test/dev purposes?  How do you test the scripts?

Reply:
The tests must be run with `njs`, which is the CLI for the NGINX JS module: http://nginx.org/en/docs/njs/cli.html.

The CLI is available along with the NGINX JS module, that's installed in the Hawtio online container image, so I run the test from there, by running:

```
$ docker run --rm -it -v `pwd`:/test hawtio/hawtio-online bash
$ cd /test/docker
$ njs test.js
```

The `test.js` file imports the `nginx.js` file as an ES module. Unfortunately, until https://github.com/nginx/njs/issues/115 gets fixed, the main script cannot be an ES module. So I have to change `function proxyJolokiaAgent(req)` to `export default function proxyJolokiaAgent(req)` so that the test module can import the main script :(

I've mostly used the tests to support the development. It's clear running the tests should be streamlined, so that these can ultimately be used for continuous testing.

Reply:
I've just found out that the latest release of NJS has introduced the `js_import` directive. So that should be possible to fix the issue. I'm trying right now... I'll keep you posted.

Reply:
I've just pushed 5ccf2ea9206a0546ae48ce321659f404a6828ade so that no manual modifications is required anymore. You can just run the tests with `njs test.js`.